tests/contrib/logbook/test_logbook_logging.py

```diff
@@ -49,6 +52,8 @@ def test_log_trace_global_values():
     Check trace info includes global values over local span values
     """
 
+    handler.records.clear()
```

Comment:
nit: this can be a fixture that's auto applied to all logbook tests

Reply:
Done